src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import useResetFocus from 'src/utils/accessibility/useResetFocus';
+
+import OffClickWrapper from '../offClickWrapper';
+import ChevronIcon from '../../svgs/icons/chevron-icon.svg';
+import {
+  DropdownContainer,
+  dropdownInputStyle,
+  IconContainer,
+  DropdownOptionsContainer,
+  DropdownOption,
+} from './style';
+
```

Comment:
File CHANGELOG:

1. Convert to TS
2. Add `useResetFocus` hook to move focus back to the button when an option is selected
3. Update roles, tabIndex, and aria-labels for the `DropdownOptionsContainer` ul and `DropdownOption` li. 
4. Add a keydown handler to allow selecting an option using the `enter` key.